cogs/onmyoji.py

```diff
@@ -1,18 +1,110 @@
+from __future__ import print_function
 import discord
 import json
 from discord.ext import commands
+import os.path
+from googleapiclient.discovery import build
+from googleapiclient.http import *
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from google.oauth2 import service_account
+import io
+import csv
+import config
+import re
 
+permission = 'You do not have permission to use this command.'
+owner_list = config.owner_list
+editor_list = config.editor_list
 
-with open('lists/bounty.json') as file:
-    bounty_json = json.loads(file.read())
+recommend = re.compile(r'recommend',re.IGNORECASE)
 
 with open('lists/stats.json') as file:
     stats_json = json.loads(file.read())
 
+def bold(input):
+    '''Returns the Discord bolded version of input text.'''
+    return "**"+input+"**"
+
+class DriveAPI:
+    '''Manages the Google Drive API Auth and retreiving the CSV database.'''
+    SCOPES = ['https://www.googleapis.com/auth/drive.readonly']
```

Comment:
IMO if its all caps, and probably not going to change often should live in `config` or `constants`

Reply:
Depending on need SCOPES would change, since in this case im not pushing and just reading, drive.readonly suffices. But I could just do SCOPES_READONLY in the config, you're right.